catalog/tests/dags/common/sensors/test_single_run_external_dags_sensor.py

```diff
@@ -216,26 +142,127 @@ def test_retries_if_running_dags_with_completed_sensor_task(self):
-            assert (
-                "INFO:airflow.task.operators.common.sensors.single_run_external"
-                "_dags_sensor.SingleRunExternalDAGsSensor:1 DAGs are in the "
-                "running state" in sensor_logs.output
-            )
+        run_sensor(sensor)
+
+
+@pytest.mark.filterwarnings(
+    "ignore:This class is deprecated. Please use "
+    "`airflow.utils.task_group.TaskGroup`.:airflow.exceptions.RemovedInAirflow3Warning"
+)
+# This also appears to be coming from Airflow internals during testing as a result of
+# loading the example bash operator DAG:
+# /home/airflow/.local/lib/python3.10/site-packages/airflow/models/dag.py:3492: RemovedInAirflow3Warning  # noqa: E501
+@pytest.mark.filterwarnings(
+    "ignore:Param `schedule_interval` is deprecated and will be removed in a future release. "
+    "Please use `schedule` instead.:airflow.exceptions.RemovedInAirflow3Warning"
+)
+def test_succeeds_if_no_running_dags(
+    caplog, sample_dag_id_fixture, sample_pool_fixture, clean_db, setup_pool
+):
+    # Create some DAGs that are not considered 'running'
+    successful_dag = create_dag(
+        "successful_dag", sample_dag_id_fixture, sample_pool_fixture
+    )
+    create_dagrun(successful_dag, State.SUCCESS)
+    failed_dag = create_dag("failed_dag", sample_dag_id_fixture, sample_pool_fixture)
+    create_dagrun(failed_dag, State.FAILED)
+
+    # DAG in the running state, but its wait task has not been started
+    queued_dag = create_dag("queued_dag", sample_dag_id_fixture, sample_pool_fixture)
+    create_dagrun(queued_dag, State.RUNNING)
+
+    # Create the Test DAG and sensor with dependent dag Ids
+    dag = DAG(
+        "test_dag_success",
+        default_args={
+            "owner": "airflow",
+            "start_date": DEFAULT_DATE,
+        },
+    )
+    sensor = SingleRunExternalDAGsSensor(
+        task_id=TEST_TASK_ID,
+        external_dag_ids=["successful_dag", "failed_dag", "queued_dag"],
+        poke_interval=5,
+        mode="reschedule",
+        dag=dag,
+        pool=sample_pool_fixture,
+    )
+    with caplog.at_level(logging.INFO):
+        run_sensor(sensor)
+        assert (
+            "Poking for DAGs ['successful_dag', 'failed_dag', 'queued_dag'] ..."
+            in caplog.text
+        )
+        assert "0 DAGs are in the running state" in caplog.text
+
+
+@pytest.mark.filterwarnings(
+    "ignore:This class is deprecated. Please use "
+    "`airflow.utils.task_group.TaskGroup`.:airflow.exceptions.RemovedInAirflow3Warning"
+)
+# This also appears to be coming from Airflow internals during testing as a result of
+# loading the example bash operator DAG:
+# /home/airflow/.local/lib/python3.10/site-packages/airflow/models/dag.py:3492: RemovedInAirflow3Warning  # noqa: E501
+@pytest.mark.filterwarnings(
+    "ignore:Param `schedule_interval` is deprecated and will be removed in a future release. "
+    "Please use `schedule` instead.:airflow.exceptions.RemovedInAirflow3Warning"
+)
```

Comment:
Just want to ping @WordPress/openverse-catalog on this to make sure this is a known deprecation. I couldn't find an issue for resolving it.

Reply:
Both warnings are being thrown by loading Airflow's example DAGs for use in the tests. It looks like the second was resolved on their end -- I just removed the filter and confirmed it's not being raised locally. The TaskGroup warning is still raised.

```suggestion
@pytest.mark.filterwarnings(
    "ignore:This class is deprecated. Please use "
    "`airflow.utils.task_group.TaskGroup`.:airflow.exceptions.RemovedInAirflow3Warning"
)
```

Reply:
Thank you @sarayourfriend and @stacimc for looking into both warnings